### ISSUE TITLE
/PageMode /UseAttachments should be honored

### ISSUE THREAD

**Issue by ousia** [mozilla/pdf.js#8657]
[This uncompressed PDF document](https://github.com/mozilla/pdf.js/files/1151246/attachment-label.pdf) is automatically displayed by _Acrobat Reader_ (old version, the latest available for _Linux_) such as in:

![attachment-open](https://user-images.githubusercontent.com/203767/28250308-5a5ea08e-6a65-11e7-8e0f-f949393999b3.png)

The attachment pane is automatically opened by this code in _Acrobat_:

```
52 0 obj
<<
  /Type /Catalog
  /Pages 24 0 R
  /Metadata 25 0 R
  /Names 27 0 R
  /PageLabels <<
    /Nums [ 0 <<
          /S /D
          /St 1
        >> ]
  >>
  /PageMode /UseAttachments
  /Version /1.7
>>
endobj
```

Possible [values for `/PageMode`](https://wwwimages2.adobe.com/content/dam/Adobe/en/devnet/pdf/pdfs/PDF32000_2008.pdf#page=82) are:

* `/UseNone`
* `/UseOutlines`
* `/UseThumbs`
* `/FullScreen`
* `/UseOC`
* `/UseAttachments`

I’m not sure whether `/FullScreen` or `/UseOutlines` are implemented, but `/UseAttachments` is not. This page mode contains essential essential information (attachments that the user might not notice as part of the PDF document).

Many thanks for your help.

**Comment by ousia** [comment]
@timvandermeij, sorry for bothering with this issue.

Is there any way to check which PDF.js version number Firefox has installed.

I’m using Firefox 56 and I don’t know whether it comes with the latest PDF.js version or not.

Many thanks for your help.

**Comment by timvandermeij** [comment]
The fix is included in version 1.9.426 and later, which is the current stable release. It looks like version 1.8.618 is included in Firefox 56, so that does not contain the fix yet. I think it will be in Firefox 57 looking at https://bugzilla.mozilla.org/show_bug.cgi?id=1398347 (which contains a more recent version that was merged for Firefox 57).
